ui/src/routes/lectures.tsx

```diff
@@ -15,6 +15,8 @@ function Lectures() {
 
   const [lectures, setLectures] = React.useState<any>();
 
+  window.scrollTo(0, 0);
```

Comment:
why do we need this?

Reply:
Because it should bring the user to the top of the page.

Reply:
When it opens a new page or triggers a new route. For example, if the user clicks on a video, the new page with the video should start from the top of the page, not in the middle or at the last position where the user left.